src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -293,23 +298,44 @@ const HealthCardDisplay = () => {
 
   return (
     <Grid container className={styles.root}>
-      {qrError ? (
-        <ErrorFallback error={qrError} />
-      ) : (
-        <ErrorBoundary FallbackComponent={ErrorFallback}>
-          <HealthCardDataProvider healthCardJws={jws}>
-            <Banners />
+      {(qrError)
+        ? (
+          <ErrorFallback error={qrError} />
+        ) : (
+          <ErrorBoundary FallbackComponent={ErrorFallback}>
+            <HealthCardDataProvider healthCardJws={jws[jws.length - 1]}>
+              <Banners />
+            </HealthCardDataProvider>
             <Grid item className={styles.flexCard}>
+              <Box>
+                <Router>
+                  <Switch>
+                    <Link
+                      to="/qr-scan"
+                      state="link"
+                      replace
+                      onClick={() => handleScan('link')}
+                    >
+                      {t(
+                        'healthcarddisplay.Add another SMART Health Card for same person'
+                      )}
+                    </Link>
+                  </Switch>
+                </Router>
+              </Box>
               {bannersUpdated && (
                 <>
-                  <VaccineCard padding="1rem" width="100%" />
-                  <QrScanButton onClick={handleScan} styles={{ padding: '1rem', width: '100%' }} />
+                  {jws.map((hcJws) => (
```

Comment:
These should display in reverse order with the most recent on top, since the banner applies to that one.